.doc_gen/cross-content/cross_detect_labels_rust_block.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+    <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+    %phrases-shared;
+]>
+<block>
+  <para>
+    Gets EXIF information from a a JPG, JPEG, or PNG file,
+    uploads the image file to an &S3long; (&S3;) bucket,
+    uses &REKlong; (&REK;) to identify the three top attributes (labels in &REK;) in the file,
```

Comment:
The word "labels" needs emphasis (italic).

Reply:
Done.